tests/suites/test_suite_nist_kw.function

```diff
@@ -165,12 +165,13 @@ void nist_kw_plaintext_lengths(int in_len, int out_len, int mode, int res)
         TEST_ASSERT(ciphertext != NULL);
     }
 
-    TEST_ASSERT(mbedtls_nist_kw_setkey(&ctx, MBEDTLS_CIPHER_ID_AES,
-                                       key, 8 * sizeof(key), 1) == 0);
+    PSA_ASSERT(setup_psa_key(PSA_KEY_USAGE_ENCRYPT | PSA_KEY_USAGE_DECRYPT, PSA_ALG_ECB_NO_PADDING,
+                             PSA_KEY_TYPE_AES, &key_id, &key));
 
-    TEST_ASSERT(mbedtls_nist_kw_wrap(&ctx, mode, plaintext, in_len,
-                                     ciphertext, &output_len,
-                                     output_len) == res);
+    TEST_ASSERT(mbedtls_nist_kw_wrap(key_id, mode,
+                                     plaintext, in_len,
+                                     ciphertext, output_len,
+                                     &output_len) == res);
```

Comment:
In new code, we prefer `TEST_EQUAL(…, res)` which shows the values on both sides if the test fails. We normally don't rewrite old code when making minor changes like you're doing here (with respect to this part of the test code), but feel free to change to `TEST_EQUAL` if it helps you diagnose test failures.